rllib/env/wrappers/pettingzoo_env.py

```diff
@@ -73,70 +80,79 @@ def __init__(self, env):
         super().__init__()
         self.env = env
         env.reset()
-        # TODO (avnishn): Remove this after making petting zoo env compatible with
```

Comment:
Seems no longer needed :)
I think we fixed the `self.action_space` might be space for all agents vs `self.action_space` is Dict mapping agent IDs to individual spaces conundrum already.

Reply:
nice